test/unit/busterbunny-raiseEvents-input-validation-tests.js

```diff
@@ -0,0 +1,92 @@
+describe('busterbunny.js raiseEvents', function() {
+    var assert = require('assert');
+    var expect = require('chai').expect;
+
+    var fakeConfig = {
+        cluster: {
+            host: 'host.host.it',
+            port: 5672,
+            vhost: '/',
+            login: 'someguy',
+            password: '2insecure',
+            heartbeat: 10
+        },
+        queues: [
+            {
+                name: 'i.read.from.this1'
+            },
+            {
+                name: 'i.read.from.this2'
+            }
+        ],
+        exchange: 'i.write.2.this1'
+    };
+
+    it('should throw an Error when raiseEvents is called and both options and afterRaised are not specified', function() {
+        var BusterBunny = require('../../src/busterbunny.js');
+        var bb = new BusterBunny(fakeConfig);
+
+        function doIt() {
+            bb.raiseEvents('eventId', {});
+        }
+
+        expect(doIt).to.throw(Error);
```

Comment:
I would rather validate specific error types, but I'll check the string messages for now.